### PR TITLE
RzShell: convert o- and oL commands

### PR DESCRIPTION
**Pull request by ret2libc**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

* Remove o-*h because it was the same as o--
* Remove o-! because not used

I'm going to do the conversion to rzshell from the bottom up, so that PRs can be smaller and I don't get stack in converting the whole `o` in one shot.

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

**Comment by wargio** [comment]
@ret2libc i saw that you have removed a test file, is that used for those removed commands only?

**Comment by ret2libc** [comment]
> @ret2libc i saw that you have removed a test file, is that used for those removed commands only?

I wrote the reason in the commit msg:
```
* Remove unread test case because it was not doing anything (o-fd did
  not work)
```

File: librz/core/cmd/cmd_open.c [hunk 1]
```diff
@@ -1625,3 +1562,41 @@ RZ_IPI int rz_cmd_open(void *data, const char *input) {
...
+
+	// TODO: Move to a-- ?
+	rz_analysis_purge(core->analysis);
+	// TODO: Move to f-- ?
+	rz_flag_unset_all(core->flags);
+	RZ_LOG_INFO("Close all files\n");
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_open_plugins_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *state) {
```
Comment (XVilka) [inline librz/core/cmd/cmd_open.c]: Isn't this a duplicate of the `L` commands?

File: librz/core/cmd_descs/cmd_open.yaml [hunk 2]
```diff
@@ -3,30 +3,26 @@
...
+    cname: open_close
+    summary: Close file descriptor
+    args:
+      - name: fd
+        type: RZ_CMD_ARG_TYPE_NUM
+  - name: o--
+    cname: open_close_all
+    summary: Close all files
+    args: []
+  - name: oL
```
Comment (XVilka) [inline librz/core/cmd_descs/cmd_open.yaml:16]: Same, a possible duplicate of one of the `L` commands.

**Review by XVilka — CHANGES_REQUESTED** [review]
.

File: librz/core/cmd/cmd_open.c (hunk 1 above)
Comment (ret2libc) [reply librz/core/cmd/cmd_open.c]: Good point, yes i think so. I will review this

**Review by XVilka — APPROVED** [review]

**Comment by XVilka** [comment]
Fix one failing test please:
```

[XX] db/cmd/cmd_json empty irj should print empty array
RZ_NOPLUGINS=1 rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc irj =
-- stdout
--- expected
+++ actual
@@ -1,1 +1,0 @@
-

-- stderr
WARNING: Could not get relocations for current bin file.
```

**Pull request merged**